mantle/platform/qemu.go

```diff
@@ -529,40 +526,33 @@ func (builder *QemuBuilder) Append(args ...string) {
 
 // baseQemuArgs takes a board and returns the basic qemu
 // arguments needed for the current architecture.
-func baseQemuArgs(board string) []string {
-	combo := runtime.GOARCH + "--" + board
-	switch combo {
-	case "amd64--amd64-usr":
+func baseQemuArgs() []string {
```

Comment:
(Arguably this one should take a host/guest combo still, but we can circle back to that if someone eventually cares)